### ISSUE TITLE
TTY phone numbers

### ISSUE THREAD

**Issue by cjerdonek** [votinginfoproject/vip-specification#242]
I was helping someone implement ElectionAdministration for San Francisco, and I noticed that there doesn't seem to be a way to designate a TTY phone number.  See SF's "Contact Us" page [here](http://www.sfgov2.org/index.aspx?page=3335) for an example of a TTY phone number.

Currently, ContactInformation just has phone number elements for "Fax" and general "Phone," but no way to designate a phone number as TTY:

```xml
<xs:complexType name="ContactInformation">
  <xs:sequence>
    <xs:element name="AddressLine" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
    <xs:element name="Email" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
    <xs:element name="Fax" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
    <xs:element name="Hours" type="InternationalizedText" minOccurs="0" />
    <xs:element name="HoursOpenId" type="xs:IDREF" minOccurs="0" />
    <xs:element name="Name" type="xs:string" minOccurs="0" />
    <xs:element name="Phone" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
    <xs:element name="Uri" type="xs:anyURI" minOccurs="0" maxOccurs="unbounded" />
  </xs:sequence>
  <xs:attribute name="identifier" type="xs:string" use="required" />
</xs:complexType>
```

Also, if you list multiple "Phone" elements (e.g. a second number for TTY), there doesn't seem to be a way to indicate how the second phone number is different.

**Comment by jdmgoogle** [comment]
Thoughts, @jktomer @jungshadow @pstenbjorn?

**Comment by cjerdonek** [comment]
Here is another data point (from [CA SOS web site](http://www.sos.ca.gov/administration/contact-information#elections)):

    Phone Numbers
    Main Number: (916) 657-2166
    TTY/TDD: (800) 833-8683
    Voter Hotline: (800) 345-VOTE (8683)
    Elections Division Fax: (916) 653-3214

**Comment by jktomer** [comment]
This is good to support. We should add a TDDNumber to ContactInformation (best not to wade into the quagmire of making things more generic).

**Comment by jdmgoogle** [comment]
Closed by #261.
